New entries in UI and UX -- fixing PR (#4)
js6450/docs: content/uiux/concepts/accessibility/accessibility.md

---
Title: 'Accessibility'
Description: 'Accessibility refers to designing devices, products, and environments such that individuals with disabilities or sensory impairments can successfully use the device or product.'
Subjects:
  - 'Web Design'
Tags:
  - 'UI'
  - 'UX'
CatalogContent:
  - 'intro-to-ui-ux'
  - 'paths/front-end-engineer-career-path'
---

_Accessibility_ refers to designing devices, products, and environments such that individuals with disabilities or sensory impairments can successfully use the device or product.

The Web Content Accessibility Guidelines (WCAG) break down accessibility into [four main principles](https://www.w3.org/WAI/standards-guidelines/wcag/glance/):

> - **Perceivable**: Create content that can be perceived in different ways, for example, providing text alternatives for non-text content and content that can be perceived by assistive technologies like screen readers.

Reviewer: While I think that bold-formatting should mostly be used when referring to a file or a folder/directory, I think this makes sense because it's showcasing the four (4) primary points from the source material. That's just me, though. 😄

Reviewer 2: Hey Brandon! This [markdown style standard](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) is actually something that changed a couple of months ago. Filenames now are supposed to use inline code (\`), i.e. `fileName.jpg`

Reviewer: Oh nice! Thanks for the clarification 😄 Gonna read up on this

---
Title: 'Accessibility'

> Description: 'Accessibility refers to designing devices, products, and environments such that individuals with disabilities or sensory impairments can successfully use the device or product.'

Reviewer 3: ```suggestion
Description: 'Accessibility refers to designing devices, products, and environments such that individuals with disabilities or sensory impairments can successfully use them.'
```

Subjects:
  - 'Web Design'
Tags:
  - 'UI'
  - 'UX'
CatalogContent:
  - 'intro-to-ui-ux'
  - 'paths/front-end-engineer-career-path'
---

<link rel="canonical" href="https://www.codecademy.com/article/what-is-digital-accessibility" />

> **Accessibility** refers to designing devices, products, and environments such that individuals with disabilities or sensory impairments can successfully use the device or product.

Reviewer 3: ```suggestion
**Accessibility** refers to designing devices, products, and environments such that individuals with disabilities or sensory impairments can successfully use them.
```

## Accessibility Guidelines

The Web Content Accessibility Guidelines (WCAG) break down accessibility into [four main principles](https://www.w3.org/WAI/standards-guidelines/wcag/glance/):

- **Perceivable**: Create content that can be perceived in different ways, for example, providing text alternatives for non-text content and content that can be perceived by assistive technologies like screen readers.

> - **Operable**: Enable simple operation of the website by making all functionality available from the keyboard, helping users navigate and find content, and giving users plenty of time to consume content.

Reviewer 3: ```suggestion
- **Operable**: Enable simple operation by making all functionality available from the keyboard, helping users navigate and find content, and giving users plenty of time to consume content.
```

Reviewer 3: The other entries implied more general applicability.

- **Understandable**: Make text readable and understandable, make content appear in predictable ways, and help users avoid and correct mistakes.
- **Robust**: Maximize compatibility with current and future user tools.

[Inclusive design](https://www.codecademy.com/resources/docs/uiux/inclusive-design) focuses on designing for users of all demographics and abilities, while accessibility is specifically focused on designing for people with disabilities. Any product that doesn't consider accessibility and equity may (whether intentionally or not) exclude a portion of the population who would otherwise find it useful.

> Learn more about accessibility and design:

Reviewer 3: ```suggestion
## Accessibility and Design Resources
```

Reviewer: If we go with a heading, let's remove the `:` (no punctuation unless the heading is phrased as a question)

Reviewer 3: Edited.
